interfaces/builtin/dvb_video.go

```diff
@@ -0,0 +1,54 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const dvbVideoSummary = `allows access to all digital video broadcasting devices`
+
+const dvbVideoBaseDeclarationSlots = `
+  dvb-video:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const dvbVideoConnectedPlugAppArmor = `
+# Until we have proper device assignment, allow access to all dvb video devices
+/dev/dvb/adapter[0-9]*/ca[0-9]* rw,
+/dev/dvb/adapter[0-9]*/demux[0-9]* rw,
+/dev/dvb/adapter[0-9]*/dvr[0-9]* r,
+/dev/dvb/adapter[0-9]*/frontend[0-9]* rw,
+/dev/dvb/adapter[0-9]*/video[0-9]* rw,
```

Comment:
Considering this should cover everything and not just video, please use:
```
# Allow access to all DVB devices
/dev/dvb/adapter[0-9]*/* rw,
# From https://www.kernel.org/doc/Documentation/admin-guide/devices.txt
/run/udev/data/c212:[0-9]* r,
```

We may need some /sys/devices accesses, but I'd need to see representative denials to recommend what else to add. This can happen in a follow-up PR if needed.

Reply:
Done